en/config/autotune.md

```diff
@@ -0,0 +1,223 @@
+However unless the configuration precisely matches your hardware you should tune the rate and attitude controllers.
+Tuning the velocity and position controllers is less important because they are less affected by vehicle dynamics, and the default tuning configuration for a similar airframe is often sufficient.
+
+Autotuning provides an automatic mechanism to tune the rate and attitude controllers.
+It can be used to tune fixed wing and multicopter vehicles, and VTOL vehicles when flying as a multicopter or as a fixed wing (transition between modes must be manually tuned).
+In theory it should work for other vehicle types that have a rate controller, but currently only the above types are supported.
+
+Automatic tuning works well for the multicopter and fixed wing vehicle configurations supported by PX4, provided the frame is not too flexible
+(see [below for more information](#does-autotuning-work-for-all-supported-airframes)).
+
+The vehicle must be flying in [Hold mode](../flight_modes/hold.md), [Position mode](../flight_modes/position_mc.md), or [altitude mode](../flight_modes/altitude_mc.md).
+The flight stack will apply a small disturbance to the vehicle in each axis and then attempt to calculate the new tuning parameters.
+For fixed wing vehicles the new tuning is applied in-air by default, after which the vehicle tests the new settings and reverts the tuning if the controllers are not stable.
+For multicopter, the vehicle lands and applies the new tuning parameters after disarming; the pilot is expected to then take off carefully and test the tuning.
+
+The tuning process takes about 40 seconds ([between 19 and 68 seconds](#how-long-does-autotuning-take)).
+The default behaviour can be configured using [parameters](#parameters).
+
+:::note
+For instructions on using the manual PID tuning see:
+- [Multicopter PID Tuning Guide](../config_mc/pid_tuning_guide_multicopter_basic.md) (Manual/Simple)
+- [Multicopter PID Tuning Guide](../config_mc/pid_tuning_guide_multicopter.md) (Advanced/Detailed)
+- [Fixed-Wing PID Tuning Guide](../config_fw/pid_tuning_guide_fixedwing.md)
+:::
+
+<!-- 
+Overview of the auto-tuning workflow
+
+@[youtube](https://youtu.be/uHVlIIydcpg)
+-->
+
+## Pre-tuning Test
+
+The vehicle must be able to fly and adequately stabilize itself before running auto-tune.
+
+:::note
+During [Airframe Setup](../config/airframe.md) you should have selected the frame that most closely matches your vehicle.
+This will usually be tuned well enough to fly, and it _may_ also be sufficiently well tuned to run autotuning.
+:::
+
+To make sure the vehicle is stable enough for auto-tuning:
+
+1. Perform a normal preflight safety checklist to ensure the flight zone is clear and has enough space.
+2. Takeoff and prepare for the test
+   - **Multicopters:** Take off and hover at 1m above ground in Position mode, Altitude mode or Hold mode.
```

Comment:
```suggestion
   - **Multicopters:** Take off and hover at 1m above ground in Altitude or Stabilized mode.
```

Those tests are mainly to see if the drone can fly safely in position in auto modes by checking the attitude stability so they need to be performed in stabilized or altitude modes.